otto/internal/app.go

```diff
@@ -5,49 +5,54 @@ package internal
 import (
 	"context"
 	"database/sql"
+	"fmt"
 	"log/slog"
 	"sync"
 
 	"github.com/google/go-github/v71/github"
+	"github.com/jferrl/go-githubauth"
+	"github.com/open-telemetry/sig-project-infra/otto/internal/config"
```

Comment:
do we have a golangci-lint config yet? The import order for this rn is non-standard.